package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@serverless-guru/logger",
-    "version": "1.0.7",
+    "version": "1.0.8",
```

Comment:
Comment for another PR.
This should be automated. 

https://github.com/semantic-release/semantic-release